Doc/library/warnings.rst

```diff
@@ -91,6 +91,9 @@ following warnings category classes are currently defined:
 | :exc:`UnicodeWarning`            | Base category for warnings related to         |
 |                                  | Unicode.                                      |
 +----------------------------------+-----------------------------------------------+
+| :exc:`BytesWarning`              | Base category for warnings related to         |
+|                                  | str and bytearray.                            |
```

Comment:
I would keep `bytes`

Reply:
Actually both str and bytes are not related. Keep just bytearray. Fix also the description in the other place.